diff -ruN ldt_orig/10DATA.LDT ldt_raw/10DATA.LDT
--- ldt_orig/10DATA.LDT	2009-08-05 16:31:00.000000000 -0500
+++ ldt_raw/10DATA.LDT	2021-11-02 11:36:16.000000000 -0500
@@ -1,5 +1,5 @@
 Univ,Time,Date,Subject,DOB,Education
-3,12:50:20,07-10-2001,10,11/20/77,15
+3,12:50:20,07-10-2001,270,11/20/77,15
 1,35092,1,1,596,supervisor's
 2,13060,1,1,613,factory
 3,14386,1,1,784,fragmentary
@@ -2003,7 +2003,7 @@
 
 
 Univ,Time,Date,Subject,DOB,Education
-3,12:01:32,07-12-2001,10,11/20/77,15
+3,12:01:32,07-12-2001,270,11/20/77,15
 2001,22174,0,1,947,methodicol
 2002,8109,1,1,684,cousin's
 2003,8611,1,1,953,curb
@@ -3382,7 +3382,7 @@
 
 ========================================
 Subject,Gender,Task,MEQ,Time,Date
-10,m,LDT,52.5,13:32:39,07-12-2001
+270,m,LDT,52.5,13:32:39,07-12-2001
 
 numCorrect,rawScore,vocabAge,shipTime,readTime
 31,33,18.2,4,2
diff -ruN ldt_orig/11DATA.LDT ldt_raw/11DATA.LDT
--- ldt_orig/11DATA.LDT	2009-08-05 16:31:00.000000000 -0500
+++ ldt_raw/11DATA.LDT	2021-11-02 11:38:26.000000000 -0500
@@ -1,5 +1,5 @@
 Univ,Time,Date,Subject,DOB,Education
-3,12:39:13,07-13-2001,11,12/29/80,14
+3,12:39:13,07-13-2001,271,12/29/80,14
 1,12925,1,1,530,extinct
 2,33094,1,1,1017,sniper's
 3,37451,1,0,810,tweet
@@ -2003,7 +2003,7 @@
 
 
 Univ,Time,Date,Subject,DOB,Education
-3,12:39:40,07-16-2001,11,12/29/80,14
+3,12:39:40,07-16-2001,271,12/29/80,14
 2001,31145,0,1,661,sman
 2002,33521,0,1,717,sceak
 2003,28989,1,1,583,reconciled
@@ -3382,7 +3382,7 @@
 
 ========================================
 Subject,Gender,Task,MEQ,Time,Date
-11,f,LDT,54,14:03:55,07-16-2001
+271,f,LDT,54,14:03:55,07-16-2001
 
 numCorrect,rawScore,vocabAge,shipTime,readTime
 27,30,17,4,2
diff -ruN ldt_orig/12DATA.LDT ldt_raw/12DATA.LDT
--- ldt_orig/12DATA.LDT	2009-08-05 16:31:00.000000000 -0500
+++ ldt_raw/12DATA.LDT	2021-11-02 11:39:24.000000000 -0500
@@ -1,5 +1,5 @@
 Univ,Time,Date,Subject,DOB,Education
-3,10:24:37,07-16-2001,12,11/30/80,14
+3,10:24:37,07-16-2001,272,11/30/80,14
 1,7676,1,1,2410,controversialists
 2,33139,1,1,793,snowstorm
 3,924,0,1,2015,Atgeria
@@ -2003,7 +2003,7 @@
 
 
 Univ,Time,Date,Subject,DOB,Education
-3,10:24:14,07-17-2001,12,11/30/80,14
+3,10:24:14,07-17-2001,272,11/30/80,14
 2001,12111,0,1,750,eftirety
 2002,29436,0,1,572,relentloss
 2003,24480,0,1,1410,oddonent
@@ -3382,7 +3382,7 @@
 
 ========================================
 Subject,Gender,Task,MEQ,Time,Date
-12,f,LDT,57,11:47:15,07-17-2001
+272,f,LDT,57,11:47:15,07-17-2001
 
 numCorrect,rawScore,vocabAge,shipTime,readTime
 34,35,19,3,2
diff -ruN ldt_orig/13DATA.LDT ldt_raw/13DATA.LDT
--- ldt_orig/13DATA.LDT	2009-08-05 16:31:00.000000000 -0500
+++ ldt_raw/13DATA.LDT	2021-11-02 11:40:16.000000000 -0500
@@ -1,5 +1,5 @@
 Univ,Time,Date,Subject,DOB,Education
-3,12:38:16,07-17-2001,13,09/26/80,14
+3,12:38:16,07-17-2001,273,09/26/80,14
 1,21226,0,1,2468,sachine
 2,12357,1,1,889,essay
 3,15558,0,1,1565,crammar
@@ -2003,7 +2003,7 @@
 
 
 Univ,Time,Date,Subject,DOB,Education
-3,09:09:35,07-18-2001,13,09/26/80,14
+3,09:09:35,07-18-2001,273,09/26/80,14
 2001,38782,0,1,785,benereal
 2002,7528,0,1,851,tonsulting
 2003,33116,0,1,597,glorkel
@@ -3382,7 +3382,7 @@
 
 ========================================
 Subject,Gender,Task,MEQ,Time,Date
-13,f,LDT,52,10:29:44,07-18-2001
+273,f,LDT,52,10:29:44,07-18-2001
 
 numCorrect,rawScore,vocabAge,shipTime,readTime
 29,31,17.4,2,15
diff -ruN ldt_orig/14DATA.LDT ldt_raw/14DATA.LDT
--- ldt_orig/14DATA.LDT	2009-08-05 16:31:00.000000000 -0500
+++ ldt_raw/14DATA.LDT	2021-11-08 09:24:15.000000000 -0600
@@ -1,5 +1,5 @@
 Univ,Time,Date,Subject,DOB,Education
-3,12:44:06,07-18-2001,14,01-16-79,17
+3,12:44:06,07-18-2001,274,01/16/79,17
 1,597,1,1,559,advocate
 2,8721,1,1,514,cyclist
 3,7788,1,1,504,cooperation
@@ -2003,7 +2003,7 @@
 
 
 Univ,Time,Date,Subject,DOB,Education
-3,12:59:31,07-19-2001,14,01/16/79,17
+3,12:59:31,07-19-2001,274,01/16/79,17
 2001,25592,0,1,544,Putrick's
 2002,15643,1,1,571,gratified
 2003,204,1,1,742,acclimate
@@ -3382,7 +3382,7 @@
 
 ========================================
 Subject,Gender,Task,MEQ,Time,Date
-14,f,LDT,30,14:21:56,07-19-2001
+274,f,LDT,30,14:21:56,07-19-2001
 
 numCorrect,rawScore,vocabAge,shipTime,readTime
 34,35,19,4,2.5
diff -ruN ldt_orig/15DATA.LDT ldt_raw/15DATA.LDT
--- ldt_orig/15DATA.LDT	2009-08-05 16:31:00.000000000 -0500
+++ ldt_raw/15DATA.LDT	2021-11-02 11:41:49.000000000 -0500
@@ -1,5 +1,5 @@
 Univ,Time,Date,Subject,DOB,Education
-3,17:03:38,07-18-2001,15,11-19-77,16
+3,17:03:38,07-18-2001,275,11-19-77,16
 1,19561,1,1,473,Jones
 2,25771,0,1,703,celvis
 3,27876,0,1,629,blosper
@@ -2003,7 +2003,7 @@
 
 
 Univ,Time,Date,Subject,DOB,Education
-3,17:00:00,07-19-2001,15,11-19-77,16
+3,17:00:00,07-19-2001,275,11-19-77,16
 2001,12307,0,1,464,edroneously
 2002,26589,1,1,370,played
 2003,24850,1,0,593,overarch
@@ -3380,7 +3380,7 @@
 
 ========================================
 Subject,Gender,Task,MEQ,Time,Date
-15,m,LDT,48,18:19:02,07-19-2001
+275,m,LDT,48,18:19:02,07-19-2001
 
 numCorrect,rawScore,vocabAge,shipTime,readTime
 37,37,19.8,2,2
diff -ruN ldt_orig/16DATA.LDT ldt_raw/16DATA.LDT
--- ldt_orig/16DATA.LDT	2009-08-05 16:31:00.000000000 -0500
+++ ldt_raw/16DATA.LDT	2021-11-02 11:42:39.000000000 -0500
@@ -1,5 +1,5 @@
 Univ,Time,Date,Subject,DOB,Education
-3,11:44:32,07-20-2001,16,09/08/79,16
+3,11:44:32,07-20-2001,276,09/08/79,16
 1,24957,0,0,658,overpepulated
 2,36422,1,1,776,tiebreak
 3,22651,1,1,649,moccasin
@@ -2003,7 +2003,7 @@
 
 
 Univ,Time,Date,Subject,DOB,Education
-3,13:27:59,07-23-2001,16,09/08/79,16
+3,13:27:59,07-23-2001,276,09/08/79,16
 2001,28684,0,0,694,pateable
 2002,8499,1,1,550,crusader
 2003,20913,1,1,479,loin
@@ -3382,7 +3382,7 @@
 
 ========================================
 Subject,Gender,Task,MEQ,Time,Date
-16,f,LDT,53,14:44:06,07-23-2001
+276,f,LDT,53,14:44:06,07-23-2001
 
 numCorrect,rawScore,vocabAge,shipTime,readTime
 34,35,19,3,4
diff -ruN ldt_orig/170DATA.LDT ldt_raw/170DATA.LDT
--- ldt_orig/170DATA.LDT	2009-08-05 16:31:00.000000000 -0500
+++ ldt_raw/170DATA.LDT	2021-11-08 09:07:14.000000000 -0600
@@ -1,5 +1,5 @@
 Univ,Time,Date,Subject,DOB,Education
-2,12:04:18,07-06-2001,170,04-1-1979,4
+2,12:04:18,07-06-2001,170,04-01-1979,4
 1,2546,0,1,1559,badgerung
 2,30405,1,1,736,roast
 3,845,1,1,905,airshaft
@@ -2003,7 +2003,7 @@
 
 
 Univ,Time,Date,Subject,DOB,Education
-2,13:07:18,07-12-2001,170,04-01-79,4
+2,13:07:18,07-12-2001,170,04-01-1979,4
 2001,6774,0,1,753,commank's
 2002,899,0,1,785,anehouse
 2003,12417,1,0,698,etymological
diff -ruN ldt_orig/176DATA.LDT ldt_raw/176DATA.LDT
--- ldt_orig/176DATA.LDT	2009-08-05 16:31:00.000000000 -0500
+++ ldt_raw/176DATA.LDT	2021-10-28 10:35:17.000000000 -0500
@@ -2003,10 +2003,6 @@
 
 
 Univ,Time,Date,Subject,DOB,Education
-1,12:05:02,07-17-2001,176,y01,1
-
-
-Univ,Time,Date,Subject,DOB,Education
 2,10:15:54,07-19-2001,176,08-27-1979,4
 2001,8855,1,1,1013,darkhaired
 2002,29960,0,1,952,cesults
diff -ruN ldt_orig/17DATA.LDT ldt_raw/17DATA.LDT
--- ldt_orig/17DATA.LDT	2009-08-05 16:31:00.000000000 -0500
+++ ldt_raw/17DATA.LDT	2021-11-02 11:43:13.000000000 -0500
@@ -1,5 +1,5 @@
 Univ,Time,Date,Subject,DOB,Education
-3,09:09:50,07-23-2001,17,06/01/78,15
+3,09:09:50,07-23-2001,277,06/01/78,15
 1,9279,0,0,561,delectabbe
 2,9888,0,0,631,Diano's
 3,31539,0,1,1039,recessionist
@@ -2003,7 +2003,7 @@
 
 
 Univ,Time,Date,Subject,DOB,Education
-3,09:09:03,07-24-2001,17,06/01/78,15
+3,09:09:03,07-24-2001,277,06/01/78,15
 2001,13672,0,1,715,cirstborn
 2002,10189,1,1,580,disco
 2003,20460,0,1,544,lethaligy
@@ -3382,7 +3382,7 @@
 
 ========================================
 Subject,Gender,Task,MEQ,Time,Date
-17,f,LDT,32,10:26:39,07-24-2001
+277,f,LDT,32,10:26:39,07-24-2001
 
 numCorrect,rawScore,vocabAge,shipTime,readTime
 28,31,17.4,3,3
diff -ruN ldt_orig/18DATA.LDT ldt_raw/18DATA.LDT
--- ldt_orig/18DATA.LDT	2009-08-05 16:31:00.000000000 -0500
+++ ldt_raw/18DATA.LDT	2021-11-02 11:43:44.000000000 -0500
@@ -1,5 +1,5 @@
 Univ,Time,Date,Subject,DOB,Education
-3,11:48:38,07-23-2001,18,10/18/79,15
+3,11:48:38,07-23-2001,278,10/18/79,15
 1,7570,1,1,1047,contemptuous
 2,1730,0,1,720,arbom
 3,31583,0,1,1416,pecurities
@@ -2003,7 +2003,7 @@
 
 
 Univ,Time,Date,Subject,DOB,Education
-3,11:33:15,07-24-2001,18,10/18/79,15
+3,11:33:15,07-24-2001,278,10/18/79,15
 2001,5924,1,0,521,chophouse
 2002,24039,1,1,681,numbing
 2003,37192,1,0,633,Trinidad
@@ -3382,7 +3382,7 @@
 
 ========================================
 Subject,Gender,Task,MEQ,Time,Date
-18,f,LDT,45.5,12:57:56,07-24-2001
+278,f,LDT,45.5,12:57:56,07-24-2001
 
 numCorrect,rawScore,vocabAge,shipTime,readTime
 26,29,16.6,4,2
diff -ruN ldt_orig/19DATA.LDT ldt_raw/19DATA.LDT
--- ldt_orig/19DATA.LDT	2009-08-05 16:31:00.000000000 -0500
+++ ldt_raw/19DATA.LDT	2021-11-02 11:44:27.000000000 -0500
@@ -1,5 +1,5 @@
 Univ,Time,Date,Subject,DOB,Education
-3,14:32:24,07-24-2001,19,7-28-78,16
+3,14:32:24,07-24-2001,279,7-28-78,16
 1,19331,0,0,1162,jackots
 2,35007,0,1,1051,mung
 3,11390,0,1,973,ecumenocal
@@ -2003,7 +2003,7 @@
 
 
 Univ,Time,Date,Subject,DOB,Education
-3,14:28:16,07-26-2001,19,7-28-78,16
+3,14:28:16,07-26-2001,279,7-28-78,16
 2001,25477,0,1,851,partation
 2002,20590,0,1,1978,wifts
 2003,35310,0,1,1230,speeper
@@ -3382,7 +3382,7 @@
 
 ========================================
 Subject,Gender,Task,MEQ,Time,Date
-19,m,LDT,35.5,15:49:25,07-26-2001
+279,m,LDT,35.5,15:49:25,07-26-2001
 
 numCorrect,rawScore,vocabAge,shipTime,readTime
 30,32,17.8,3,1.5
diff -ruN ldt_orig/1DATA.LDT ldt_raw/1DATA.LDT
--- ldt_orig/1DATA.LDT	2009-08-05 16:31:00.000000000 -0500
+++ ldt_raw/1DATA.LDT	2021-11-08 09:21:12.000000000 -0600
@@ -1,5 +1,5 @@
 Univ,Time,Date,Subject,DOB,Education
-3,09:14:59,06-26-2001,1,05\10\81,14
+3,09:14:59,06-26-2001,261,05/10/81,14
 1,3874,1,0,707,bookie
 2,16177,0,1,769,gandbrake
 3,26152,1,1,526,philosophical
@@ -2003,7 +2003,7 @@
 
 
 Univ,Time,Date,Subject,DOB,Education
-3,09:09:35,07-02-2001,1,05/10/81,14
+3,09:09:35,07-02-2001,261,05/10/81,14
 2001,6594,1,1,540,coinciding
 2002,37416,0,1,856,curnouts
 2003,11072,1,1,522,drown
@@ -3382,7 +3382,7 @@
 
 ========================================
 Subject,Gender,Task,MEQ,Time,Date
-1,m,LDT,34.5,10:28:53,07-02-2001
+261,m,LDT,34.5,10:28:53,07-02-2001
 
 numCorrect,rawScore,vocabAge,shipTime,readTime
 27,30,17,3,1
diff -ruN ldt_orig/20DATA.LDT ldt_raw/20DATA.LDT
--- ldt_orig/20DATA.LDT	2009-08-05 16:31:00.000000000 -0500
+++ ldt_raw/20DATA.LDT	2021-11-02 11:45:11.000000000 -0500
@@ -1,5 +1,5 @@
 Univ,Time,Date,Subject,DOB,Education
-3,09:15:22,07-25-2001,20,12/03/79,15
+3,09:15:22,07-25-2001,280,12/03/79,15
 1,3653,0,1,594,clowfly
 2,1177,0,1,762,abbunition
 3,29303,1,1,552,regrettable
@@ -2003,7 +2003,7 @@
 
 
 Univ,Time,Date,Subject,DOB,Education
-3,09:19:17,07-27-2001,20,12/03/79,15
+3,09:19:17,07-27-2001,280,12/03/79,15
 2001,28809,0,1,786,meality
 2002,16461,0,1,789,headlemp
 2003,32083,1,1,528,shelves
@@ -3380,7 +3380,7 @@
 
 ========================================
 Subject,Gender,Task,MEQ,Time,Date
-20,f,LDT,69.5,10:40:28,07-27-2001
+280,f,LDT,69.5,10:40:28,07-27-2001
 
 numCorrect,rawScore,vocabAge,shipTime,readTime
 34,35,19,2,4
diff -ruN ldt_orig/219DATA.LDT ldt_raw/219DATA.LDT
--- ldt_orig/219DATA.LDT	2009-08-05 16:31:00.000000000 -0500
+++ ldt_raw/219DATA.LDT	2021-11-08 09:12:31.000000000 -0600
@@ -1,5 +1,5 @@
 Univ,Time,Date,Subject,DOB,Education
-2,15:52:05,02-12-2002,219,09-08-1982,1
+2,15:52:05,02-12-2002,219,09-09-1982,1
 1,1844,1,1,542,armhole
 2,35959,0,1,724,tensiopless
 3,22718,0,1,1360,mald
diff -ruN ldt_orig/21DATA.LDT ldt_raw/21DATA.LDT
--- ldt_orig/21DATA.LDT	2009-08-05 16:31:00.000000000 -0500
+++ ldt_raw/21DATA.LDT	2021-11-08 09:24:43.000000000 -0600
@@ -1,5 +1,5 @@
 Univ,Time,Date,Subject,DOB,Education
-3,12:38:40,07-25-2001,21,02/16/77,15
+3,12:38:40,07-25-2001,281,2-16-77,15
 1,5749,1,1,648,cheerless
 2,36526,1,0,973,tiptoeing
 3,32518,1,1,544,simplicity
@@ -2003,7 +2003,7 @@
 
 
 Univ,Time,Date,Subject,DOB,Education
-3,12:49:42,07-27-2001,21,2-16-77,16
+3,12:49:42,07-27-2001,281,2-16-77,16
 2001,19116,0,1,486,igvite
 2002,3467,1,46,465,blacked
 2003,4059,1,1,518,boy's
@@ -3380,7 +3380,7 @@
 
 ========================================
 Subject,Gender,Task,MEQ,Time,Date
-21,m,LDT,58,14:06:30,07-27-2001
+281,m,LDT,58,14:06:30,07-27-2001
 
 numCorrect,rawScore,vocabAge,shipTime,readTime
 33,34,18.6,2,1
diff -ruN ldt_orig/22DATA.LDT ldt_raw/22DATA.LDT
--- ldt_orig/22DATA.LDT	2009-08-05 16:31:00.000000000 -0500
+++ ldt_raw/22DATA.LDT	2021-11-08 09:25:48.000000000 -0600
@@ -1,5 +1,5 @@
 Univ,Time,Date,Subject,DOB,Education
-3,15:09:15,07-25-2001,22,5-26-80,15
+3,15:09:15,07-25-2001,282,05/26/80,15
 1,8551,0,1,1171,fucumber
 2,32865,1,1,709,slippage
 3,4029,1,1,553,bowel
@@ -2003,7 +2003,7 @@
 
 
 Univ,Time,Date,Subject,DOB,Education
-3,11:39:01,07-26-2001,22,05/26/80,15
+3,11:39:01,07-26-2001,282,05/26/80,15
 2001,32677,1,1,557,skillfulness
 2002,35851,0,1,635,deleology
 2003,319,1,1,557,acquiescent
@@ -3380,7 +3380,7 @@
 
 ========================================
 Subject,Gender,Task,MEQ,Time,Date
-22,m,LDT,37.5,12:55:03,07-26-2001
+282,m,LDT,37.5,12:55:03,07-26-2001
 
 numCorrect,rawScore,vocabAge,shipTime,readTime
 34,35,19,2,2
diff -ruN ldt_orig/23DATA.LDT ldt_raw/23DATA.LDT
--- ldt_orig/23DATA.LDT	2009-08-05 16:31:00.000000000 -0500
+++ ldt_raw/23DATA.LDT	2021-11-02 11:48:04.000000000 -0500
@@ -1,5 +1,5 @@
 Univ,Time,Date,Subject,DOB,Education
-3,10:15:57,07-30-2001,23,02/20/80,15
+3,10:15:57,07-30-2001,283,02/20/80,15
 1,20639,0,1,714,limms
 2,22343,1,1,710,millstone
 3,9121,1,0,1043,decries
@@ -2003,7 +2003,7 @@
 
 
 Univ,Time,Date,Subject,DOB,Education
-3,10:17:51,07-31-2001,23,02/20/80,15
+3,10:17:51,07-31-2001,283,02/20/80,15
 2001,40140,0,1,745,worls
 2002,36377,0,1,796,Snursday
 2003,28012,1,1,712,psychic
@@ -3382,7 +3382,7 @@
 
 ========================================
 Subject,Gender,Task,MEQ,Time,Date
-23,f,LDT,30,11:46:21,07-31-2001
+283,f,LDT,30,11:46:21,07-31-2001
 
 numCorrect,rawScore,vocabAge,shipTime,readTime
 30,32,17.8,4,0
diff -ruN ldt_orig/24DATA.LDT ldt_raw/24DATA.LDT
--- ldt_orig/24DATA.LDT	2009-08-05 16:31:00.000000000 -0500
+++ ldt_raw/24DATA.LDT	2021-11-02 11:48:47.000000000 -0500
@@ -1,5 +1,5 @@
 Univ,Time,Date,Subject,DOB,Education
-3,12:40:39,07-30-2001,24,06/26/82,13
+3,12:40:39,07-30-2001,284,06/26/82,13
 1,24813,1,1,776,outskirts
 2,6545,1,0,1007,coexist
 3,4422,0,1,1220,brutolity
@@ -2003,7 +2003,7 @@
 
 
 Univ,Time,Date,Subject,DOB,Education
-3,12:37:05,07-31-2001,24,06/26/82,13
+3,12:37:05,07-31-2001,284,06/26/82,13
 2001,38391,1,1,1318,untellable
 2002,18444,1,1,1335,ingot
 2003,13231,1,1,737,fatality
@@ -3382,7 +3382,7 @@
 
 ========================================
 Subject,Gender,Task,MEQ,Time,Date
-24,f,LDT,49.5,13:59:48,07-31-2001
+284,f,LDT,49.5,13:59:48,07-31-2001
 
 numCorrect,rawScore,vocabAge,shipTime,readTime
 32,34,18.6,2,5
diff -ruN ldt_orig/253DATA.LDT ldt_raw/253DATA.LDT
--- ldt_orig/253DATA.LDT	2009-08-05 16:31:00.000000000 -0500
+++ ldt_raw/253DATA.LDT	2021-11-08 09:20:39.000000000 -0600
@@ -1,5 +1,5 @@
 Univ,Time,Date,Subject,DOB,Education
-2,15:51:59,04-24-2003,253,2-15-1982,3
+2,15:51:59,04-24-2003,253,2-15-1983,3
 1,2835,0,1,404,bact
 2,33574,0,1,554,spectromuter
 3,29567,0,1,414,rendeting
diff -ruN ldt_orig/25DATA.LDT ldt_raw/25DATA.LDT
--- ldt_orig/25DATA.LDT	2009-08-05 16:31:00.000000000 -0500
+++ ldt_raw/25DATA.LDT	2021-11-02 11:49:35.000000000 -0500
@@ -1,5 +1,5 @@
 Univ,Time,Date,Subject,DOB,Education
-3,09:11:37,08-02-2001,25,03/10/80,15
+3,09:11:37,08-02-2001,285,03/10/80,15
 1,13379,1,1,565,felony
 2,32415,0,1,811,bienna
 3,40151,1,1,552,worked
@@ -2003,7 +2003,7 @@
 
 
 Univ,Time,Date,Subject,DOB,Education
-3,09:12:26,08-03-2001,25,03/10/80,15
+3,09:12:26,08-03-2001,285,03/10/80,15
 2001,29984,0,1,552,setaining
 2002,9520,0,1,507,geposit
 2003,25060,1,1,541,Oxford
@@ -3380,7 +3380,7 @@
 
 ========================================
 Subject,Gender,Task,MEQ,Time,Date
-25,f,LDT,46.5,10:45:36,08-03-2001
+285,f,LDT,46.5,10:45:36,08-03-2001
 
 numCorrect,rawScore,vocabAge,shipTime,readTime
 23,27,15.9,5,1
diff -ruN ldt_orig/26DATA.LDT ldt_raw/26DATA.LDT
--- ldt_orig/26DATA.LDT	2009-08-05 16:31:00.000000000 -0500
+++ ldt_raw/26DATA.LDT	2021-11-02 11:50:44.000000000 -0500
@@ -1,5 +1,5 @@
 Univ,Time,Date,Subject,DOB,Education
-3,11:27:14,08-02-2001,26,09/27/79,15
+3,11:27:14,08-02-2001,286,09/27/79,15
 1,9004,1,1,667,decades
 2,36004,1,1,615,terriers
 3,22627,0,1,547,littens
@@ -2003,7 +2003,7 @@
 
 
 Univ,Time,Date,Subject,DOB,Education
-3,11:31:10,08-03-2001,26,09/27/79,15
+3,11:31:10,08-03-2001,286,09/27/79,15
 2001,34774,0,0,567,subsidizotien
 2002,13112,1,1,619,fallible
 2003,39508,0,1,689,reakest
@@ -3382,7 +3382,7 @@
 
 ========================================
 Subject,Gender,Task,MEQ,Time,Date
-26,f,LDT,32,12:46:53,08-03-2001
+286,f,LDT,32,12:46:53,08-03-2001
 
 numCorrect,rawScore,vocabAge,shipTime,readTime
 31,33,18.2,2,2
diff -ruN ldt_orig/27DATA.LDT ldt_raw/27DATA.LDT
--- ldt_orig/27DATA.LDT	2009-08-05 16:31:00.000000000 -0500
+++ ldt_raw/27DATA.LDT	2021-11-02 11:57:50.000000000 -0500
@@ -1,5 +1,5 @@
 Univ,Time,Date,Subject,DOB,Education
-3,10:18:48,08-07-2001,27,08/20/57,18
+3,10:18:48,08-07-2001,287,08/20/57,18
 1,10300,0,1,855,tisgusting
 2,279,0,1,898,aletate
 3,6896,0,1,1229,homparatively
@@ -2003,7 +2003,7 @@
 
 
 Univ,Time,Date,Subject,DOB,Education
-3,10:12:35,08-08-2001,27,08/20/57,18
+3,10:12:35,08-08-2001,287,08/20/57,18
 2001,12676,0,1,738,elerting
 2002,3424,0,1,761,dirthrate
 2003,33376,1,1,590,sore
@@ -3382,7 +3382,7 @@
 
 ========================================
 Subject,Gender,Task,MEQ,Time,Date
-27,f,LDT,45,11:30:26,08-08-2001
+287,f,LDT,45,11:30:26,08-08-2001
 
 numCorrect,rawScore,vocabAge,shipTime,readTime
 38,38,20.2,2,5
diff -ruN ldt_orig/28DATA.LDT ldt_raw/28DATA.LDT
--- ldt_orig/28DATA.LDT	2009-08-05 16:31:00.000000000 -0500
+++ ldt_raw/28DATA.LDT	2021-11-02 11:57:48.000000000 -0500
@@ -1,5 +1,5 @@
 Univ,Time,Date,Subject,DOB,Education
-3,14:39:11,08-08-2001,28,05/31/77,15
+3,14:39:11,08-08-2001,288,05/31/77,15
 1,29483,0,1,1254,gemaining
 2,4061,1,1,622,brace
 3,18237,1,1,749,Indonesian
@@ -2003,7 +2003,7 @@
 
 
 Univ,Time,Date,Subject,DOB,Education
-3,14:48:45,08-09-2001,28,05/31/77,15
+3,14:48:45,08-09-2001,288,05/31/77,15
 2001,32568,1,1,382,sink
 2002,23544,0,0,556,Depal
 2003,27324,0,1,607,creludes
@@ -3380,7 +3380,7 @@
 
 ========================================
 Subject,Gender,Task,MEQ,Time,Date
-28,m,LDT,47.5,16:04:16,08-09-2001
+288,m,LDT,47.5,16:04:16,08-09-2001
 
 numCorrect,rawScore,vocabAge,shipTime,readTime
 29,31,17.4,3,.5
diff -ruN ldt_orig/292DATA.LDT ldt_raw/292DATA.LDT
--- ldt_orig/292DATA.LDT	2009-08-05 16:31:00.000000000 -0500
+++ ldt_raw/292DATA.LDT	2021-11-08 09:29:44.000000000 -0600
@@ -1,5 +1,5 @@
 Univ,Time,Date,Subject,DOB,Education
-3,09:56:42,02-04-2002,292,kl,2
+3,09:56:42,02-04-2002,292,32580,2
 1,33314,0,1,1604,tomersault
 2,30837,0,1,2532,sagling
 3,14787,0,1,2299,Gabrielme
diff -ruN ldt_orig/2DATA.LDT ldt_raw/2DATA.LDT
--- ldt_orig/2DATA.LDT	2009-08-05 16:31:00.000000000 -0500
+++ ldt_raw/2DATA.LDT	2021-11-02 11:27:34.000000000 -0500
@@ -1,5 +1,5 @@
 Univ,Time,Date,Subject,DOB,Education
-3,11:43:31,06-26-2001,2,05/18/80,15
+3,11:43:31,06-26-2001,262,05/18/80,15
 1,18294,1,1,474,inert
 2,24203,1,1,697,obtrusive
 3,9290,0,0,1616,meliberate
@@ -2003,7 +2003,7 @@
 
 
 Univ,Time,Date,Subject,DOB,Education
-3,09:08:49,06-28-2001,2,05/18/80,15
+3,09:08:49,06-28-2001,262,05/18/80,15
 2001,31493,1,1,913,seaports
 2002,14007,1,1,901,flutter
 2003,105,0,1,782,alsolutely
@@ -3380,7 +3380,7 @@
 
 ========================================
 Subject,Gender,Task,MEQ,Time,Date
-2,f,LDT,63,10:52:00,06-28-2001
+262,f,LDT,63,10:52:00,06-28-2001
 
 numCorrect,rawScore,vocabAge,shipTime,readTime
 0,0,0,0,1
diff -ruN ldt_orig/301DATA.LDT ldt_raw/301DATA.LDT
--- ldt_orig/301DATA.LDT	2009-08-05 16:31:00.000000000 -0500
+++ ldt_raw/301DATA.LDT	2021-11-08 10:10:22.000000000 -0600
@@ -1,5 +1,5 @@
 Univ,Time,Date,Subject,DOB,Education
-3,14:37:58,02-26-2002,301,020283,14
+3,14:37:58,02-26-2002,301,022083,14
 1,24149,0,1,1053,otlivious
 2,26982,0,0,756,Hortland
 3,18463,0,1,1664,inharminious
@@ -2003,7 +2003,7 @@
 
 
 Univ,Time,Date,Subject,DOB,Education
-3,14:43:19,02-28-2002,301,202083,14
+3,14:43:19,02-28-2002,301,022083,14
 2001,24410,1,1,435,onions
 2002,18798,0,1,589,inbentioned
 2003,11722,1,1,640,embodiments
diff -ruN ldt_orig/323DATA.LDT ldt_raw/323DATA.LDT
--- ldt_orig/323DATA.LDT	2009-08-05 16:31:00.000000000 -0500
+++ ldt_raw/323DATA.LDT	2021-11-08 09:32:24.000000000 -0600
@@ -2003,7 +2003,7 @@
 
 
 Univ,Time,Date,Subject,DOB,Education
-3,11:09:48,05-08-2002,323,1-23-83,13
+3,11:09:48,05-08-2002,323,1-12-83,13
 2001,28936,0,1,506,rechock
 2002,15512,0,0,561,foverns
 2003,32867,0,0,600,slupper
diff -ruN ldt_orig/331DATA.LDT ldt_raw/331DATA.LDT
--- ldt_orig/331DATA.LDT	2009-08-05 16:31:00.000000000 -0500
+++ ldt_raw/331DATA.LDT	2021-11-08 09:35:30.000000000 -0600
@@ -2003,7 +2003,7 @@
 
 
 Univ,Time,Date,Subject,DOB,Education
-3,11:52:19,06-15-2002,331,08/06/80,16
+3,11:52:19,06-15-2002,331,08-06-1980,16
 2001,508,1,1,476,adolescents
 2002,6914,0,1,547,computriots
 2003,4587,0,1,541,bunteg
diff -ruN ldt_orig/392data.ldt ldt_raw/392data.ldt
--- ldt_orig/392data.ldt	2009-08-05 16:31:00.000000000 -0500
+++ ldt_raw/392data.ldt	2021-11-02 12:45:07.000000000 -0500
@@ -1,5 +1,5 @@
 Univ,Time,Date,Subject,DOB,Education
-4,12:47:12,03-30-2001,2,10/13/76,7
+4,12:47:12,03-30-2001,392,10/13/76,7
 1,18294,1,1,365,inert
 2,24203,1,1,451,obtrusive
 3,9290,0,0,622,meliberate
@@ -2003,7 +2003,7 @@
 
 
 Univ,Time,Date,Subject,DOB,Education
-4,16:19:36,04-02-2001,2,10/13/76,7
+4,16:19:36,04-02-2001,392,10/13/76,7
 2001,31493,1,1,696,seaports
 2002,14007,1,1,651,flutter
 2003,105,0,0,651,alsolutely
@@ -3380,7 +3380,7 @@
 
 ========================================
 Subject,Gender,Task,MEQ,Time,Date
-2,f,LDT,40,17:49:43,04-02-2001
+392,f,LDT,40,17:49:43,04-02-2001
 
 numCorrect,rawScore,vocabAge,shipTime,readTime
 35,36,19.4,6,2
diff -ruN ldt_orig/3DATA.LDT ldt_raw/3DATA.LDT
--- ldt_orig/3DATA.LDT	2009-08-05 16:31:00.000000000 -0500
+++ ldt_raw/3DATA.LDT	2021-11-02 11:28:46.000000000 -0500
@@ -1,5 +1,5 @@
 Univ,Time,Date,Subject,DOB,Education
-3,11:40:18,06-27-2001,3,02/12/82,13
+3,11:40:18,06-27-2001,263,02/12/82,13
 1,12107,0,1,559,erticements
 2,16039,1,0,367,gypsum
 3,4180,0,1,486,dreastplate
@@ -2003,7 +2003,7 @@
 
 
 Univ,Time,Date,Subject,DOB,Education
-3,11:36:21,06-28-2001,3,02/12/82,13
+3,11:36:21,06-28-2001,263,02/12/82,13
 2001,21886,0,1,540,neals
 2002,40142,1,1,503,wordy
 2003,3604,1,1,544,bloke
@@ -3380,7 +3380,7 @@
 
 ========================================
 Subject,Gender,Task,MEQ,Time,Date
-3,f,LDT,40.5,12:54:51,06-28-2001
+263,f,LDT,40.5,12:54:51,06-28-2001
 
 numCorrect,rawScore,vocabAge,shipTime,readTime
 0,0,0,0,2
diff -ruN ldt_orig/404DATA.LDT ldt_raw/404DATA.LDT
--- ldt_orig/404DATA.LDT	2009-08-05 16:31:00.000000000 -0500
+++ ldt_raw/404DATA.LDT	2021-10-28 10:24:52.000000000 -0500
@@ -3382,18 +3382,6 @@
 
 ========================================
 Subject,Gender,Task,MEQ,Time,Date
-404,f,LDT,29,12:41:21,05-29-2001
-
-numCorrect,rawScore,vocabAge,shipTime,readTime
-0,0,0,0,4
-
-presHealth,pastHealth,vision,hearing,firstLang
-5,6,5,7,English
-========================================
-
-
-========================================
-Subject,Gender,Task,MEQ,Time,Date
 404,f,LDT,61,12:59:09,05-29-2001
 
 numCorrect,rawScore,vocabAge,shipTime,readTime
diff -ruN ldt_orig/420DATA.LDT ldt_raw/420DATA.LDT
--- ldt_orig/420DATA.LDT	2009-08-05 16:31:00.000000000 -0500
+++ ldt_raw/420DATA.LDT	2021-11-01 14:20:29.000000000 -0500
@@ -2003,10 +2003,6 @@
 
 
 Univ,Time,Date,Subject,DOB,Education
-4,12:33:09,07-05-2001,420,04/17/79,5
-
-
-Univ,Time,Date,Subject,DOB,Education
 4,12:33:42,07-05-2001,420,04/17/79,5
 2001,35015,1,1,580,sunning
 2002,34187,1,1,615,stenographer
diff -ruN ldt_orig/431DATA.LDT ldt_raw/431DATA.LDT
--- ldt_orig/431DATA.LDT	2009-08-05 16:31:00.000000000 -0500
+++ ldt_raw/431DATA.LDT	2021-11-01 14:21:45.000000000 -0500
@@ -2003,10 +2003,6 @@
 
 
 Univ,Time,Date,Subject,DOB,Education
-4,12:05:37,07-27-2001,431,01/22/75,4
-
-
-Univ,Time,Date,Subject,DOB,Education
 4,12:07:49,07-27-2001,431,01/22/75,4
 2001,427,0,1,1032,acheres
 2002,16593,1,1,724,hefty
diff -ruN ldt_orig/436DATA.LDT ldt_raw/436DATA.LDT
--- ldt_orig/436DATA.LDT	2009-08-05 16:31:00.000000000 -0500
+++ ldt_raw/436DATA.LDT	2021-10-28 10:26:03.000000000 -0500
@@ -3382,18 +3382,6 @@
 
 ========================================
 Subject,Gender,Task,MEQ,Time,Date
-436,f,LDT,0,12:06:40,09-13-2001
-
-numCorrect,rawScore,vocabAge,shipTime,readTime
-0,0,0,0,2
-
-presHealth,pastHealth,vision,hearing,firstLang
-7,7,5,7,English
-========================================
-
-
-========================================
-Subject,Gender,Task,MEQ,Time,Date
 436,f,LDT,38,12:27:36,09-13-2001
 
 numCorrect,rawScore,vocabAge,shipTime,readTime
diff -ruN ldt_orig/4DATA.LDT ldt_raw/4DATA.LDT
--- ldt_orig/4DATA.LDT	2009-08-05 16:31:00.000000000 -0500
+++ ldt_raw/4DATA.LDT	2021-11-02 11:29:48.000000000 -0500
@@ -1,5 +1,5 @@
 Univ,Time,Date,Subject,DOB,Education
-3,14:11:09,06-28-2001,4,12-09-73,16
+3,14:11:09,06-28-2001,264,12-09-73,16
 1,17122,1,1,570,hopscotch
 2,26395,0,0,544,pimking
 3,22100,1,1,603,merger
@@ -2003,7 +2003,7 @@
 
 
 Univ,Time,Date,Subject,DOB,Education
-3,12:04:39,07-04-2001,4,12-09-73,16
+3,12:04:39,07-04-2001,264,12-09-73,16
 2001,28987,1,1,767,recompense
 2002,1019,0,1,754,alluped
 2003,10599,0,1,590,dippo
@@ -3382,7 +3382,7 @@
 
 ========================================
 Subject,Gender,Task,MEQ,Time,Date
-4,f,LDT,38,13:27:21,07-04-2001
+264,f,LDT,38,13:27:21,07-04-2001
 
 numCorrect,rawScore,vocabAge,shipTime,readTime
 34,35,19,4,2
diff -ruN ldt_orig/520DATA.LDT ldt_raw/520DATA.LDT
--- ldt_orig/520DATA.LDT	2009-08-05 16:31:00.000000000 -0500
+++ ldt_raw/520DATA.LDT	2021-10-28 10:27:30.000000000 -0500
@@ -3388,15 +3388,3 @@
 presHealth,pastHealth,vision,hearing,firstLang
 7,6,7,6,English
 ========================================
-
-
-========================================
-Subject,Gender,Task,MEQ,Time,Date
-520,f,LDT,54,16:05:58,03-25-2002
-
-numCorrect,rawScore,vocabAge,shipTime,readTime
-32,34,18.6,5,3
-
-presHealth,pastHealth,vision,hearing,firstLang
-7,6,7,6,English
-========================================
diff -ruN ldt_orig/5DATA.LDT ldt_raw/5DATA.LDT
--- ldt_orig/5DATA.LDT	2009-08-05 16:31:00.000000000 -0500
+++ ldt_raw/5DATA.LDT	2021-11-08 09:23:00.000000000 -0600
@@ -1,5 +1,5 @@
 Univ,Time,Date,Subject,DOB,Education
-3,15:20:07,07-02-2001,5,4-07-68,17
+3,15:20:07,07-02-2001,265,4-7-68,17
 1,10759,0,1,946,domanance
 2,29795,1,1,753,resented
 3,16828,1,1,473,hiked
@@ -2003,7 +2003,7 @@
 
 
 Univ,Time,Date,Subject,DOB,Education
-3,14:30:34,07-03-2001,5,4-7-68,17
+3,14:30:34,07-03-2001,265,4-7-68,17
 2001,28608,0,1,577,tanchers
 2002,22256,1,1,1021,midmost
 2003,40262,0,1,604,triggle
@@ -3382,7 +3382,7 @@
 
 ========================================
 Subject,Gender,Task,MEQ,Time,Date
-5,f,LDT,36.5,15:50:44,07-03-2001
+265,f,LDT,36.5,15:50:44,07-03-2001
 
 numCorrect,rawScore,vocabAge,shipTime,readTime
 40,40,21,3,2
diff -ruN ldt_orig/6DATA.LDT ldt_raw/6DATA.LDT
--- ldt_orig/6DATA.LDT	2009-08-05 16:31:00.000000000 -0500
+++ ldt_raw/6DATA.LDT	2021-11-02 11:32:01.000000000 -0500
@@ -1,5 +1,5 @@
 Univ,Time,Date,Subject,DOB,Education
-3,11:15:45,07-03-2001,6,06/17/79,16
+3,11:15:45,07-03-2001,266,06/17/79,16
 1,1831,0,1,557,atmada
 2,37903,0,1,545,urdo
 3,20741,1,1,538,list
@@ -2003,7 +2003,7 @@
 
 
 Univ,Time,Date,Subject,DOB,Education
-3,11:10:53,07-09-2001,6,06/17/79,16
+3,11:10:53,07-09-2001,266,06/17/79,16
 2001,19725,1,1,551,Kashmir
 2002,12460,1,1,626,evaluating
 2003,33485,0,1,530,blans
@@ -3382,7 +3382,7 @@
 
 ========================================
 Subject,Gender,Task,MEQ,Time,Date
-6,f,LDT,40,12:34:56,07-09-2001
+266,f,LDT,40,12:34:56,07-09-2001
 
 numCorrect,rawScore,vocabAge,shipTime,readTime
 34,35,19,4,3
diff -ruN ldt_orig/8DATA.LDT ldt_raw/8DATA.LDT
--- ldt_orig/8DATA.LDT	2009-08-05 16:31:00.000000000 -0500
+++ ldt_raw/8DATA.LDT	2021-11-08 09:23:25.000000000 -0600
@@ -1,5 +1,5 @@
 Univ,Time,Date,Subject,DOB,Education
-3,16:58:09,07-09-2001,8,2-10-82,13
+3,16:58:09,07-09-2001,268,2-10-1982,13
 1,34513,0,1,885,shresses
 2,29801,0,1,1633,peserved
 3,1503,0,0,1424,Antoite
@@ -2003,7 +2003,7 @@
 
 
 Univ,Time,Date,Subject,DOB,Education
-3,16:58:58,07-10-2001,8,2-10-1982,13
+3,16:58:58,07-10-2001,268,2-10-1982,13
 2001,20924,0,0,534,Londin's
 2002,16615,1,1,803,he'll
 2003,25222,0,1,794,Pamelu
@@ -3380,7 +3380,7 @@
 
 ========================================
 Subject,Gender,Task,MEQ,Time,Date
-8,f,LDT,62,18:22:16,07-10-2001
+268,f,LDT,62,18:22:16,07-10-2001
 
 numCorrect,rawScore,vocabAge,shipTime,readTime
 30,32,17.8,3,2.5
diff -ruN ldt_orig/9DATA.LDT ldt_raw/9DATA.LDT
--- ldt_orig/9DATA.LDT	2009-08-05 16:31:00.000000000 -0500
+++ ldt_raw/9DATA.LDT	2021-11-02 11:34:43.000000000 -0500
@@ -1,5 +1,5 @@
 Univ,Time,Date,Subject,DOB,Education
-3,09:14:48,07-10-2001,9,02/09/78,17
+3,09:14:48,07-10-2001,269,02/09/78,17
 1,5703,1,1,836,chatty
 2,10790,1,1,818,don't
 3,16266,0,1,886,tappy
@@ -2003,7 +2003,7 @@
 
 
 Univ,Time,Date,Subject,DOB,Education
-3,09:12:36,07-11-2001,9,02/09/78,17
+3,09:12:36,07-11-2001,269,02/09/78,17
 2001,39655,0,1,972,slarves
 2002,20666,0,1,981,Lindsuy
 2003,10552,0,1,1804,distortabbe
@@ -3382,7 +3382,7 @@
 
 ========================================
 Subject,Gender,Task,MEQ,Time,Date
-9,f,LDT,63,10:39:13,07-11-2001
+269,f,LDT,63,10:39:13,07-11-2001
 
 numCorrect,rawScore,vocabAge,shipTime,readTime
 37,37,19.8,2,3
diff -ruN ldt_orig/Data1065.LDT ldt_raw/Data1065.LDT
--- ldt_orig/Data1065.LDT	2009-08-05 16:31:00.000000000 -0500
+++ ldt_raw/Data1065.LDT	2021-11-03 10:34:13.000000000 -0500
@@ -2003,10 +2003,6 @@
 
 
 Univ,Time,Date,Subject,DOB,Education
-4,14:31:30,10-08-2001,455,02/21/69,3
-
-
-Univ,Time,Date,Subject,DOB,Education
 4,14:33:20,10-08-2001,455,02/21/69,3
 2001,4320,0,1,906,froader
 2002,34812,1,1,883,subsystems
diff -ruN ldt_orig/Data1634.LDT ldt_raw/Data1634.LDT
--- ldt_orig/Data1634.LDT	2009-08-05 16:31:00.000000000 -0500
+++ ldt_raw/Data1634.LDT	2021-10-28 10:28:35.000000000 -0500
@@ -3380,18 +3380,6 @@
 
 ========================================
 Subject,Gender,Task,MEQ,Time,Date
-598,f,LDT,49,16:26:20,02-18-2002
-
-numCorrect,rawScore,vocabAge,shipTime,readTime
-0,0,0,0,1
-
-presHealth,pastHealth,vision,hearing,firstLang
-7,5,7,7,English
-========================================
-
-
-========================================
-Subject,Gender,Task,MEQ,Time,Date
 598,f,LDT,49,18:09:28,02-18-2002
 
 numCorrect,rawScore,vocabAge,shipTime,readTime
diff -ruN ldt_orig/Data1988.LDT ldt_raw/Data1988.LDT
--- ldt_orig/Data1988.LDT	2009-08-05 16:31:00.000000000 -0500
+++ ldt_raw/Data1988.LDT	2021-11-02 12:01:47.000000000 -0500
@@ -1743,7 +1743,7 @@
 1741,36647,1,1,671,toner
 1742,39785,1,1,852,whitewashed
 1743,30690,1,1,581,rulers
-1744,35819,0,1,91745,23822,0,1,1278,nondryind
+1745,23822,0,1,1278,nondryind
 1746,11800,0,1,1245,emporically
 1747,4573,1,0,1305,bungle
 1748,26615,0,1,875,slea
@@ -1999,7 +1999,10 @@
 1998,40123,1,1,3374,woody
 1999,39930,1,1,1033,windfall
 2000,36692,0,1,1320,topmyst
-2000,36692,0,1,1282,topmyst
+
+
+Univ,Time,Date,Subject,DOB,Education
+6,16:07:43,03-25-2002,756,03/18/77,16
 2001,21846,0,1,1123,mazi
 2002,2162,0,1,1546,ashletes
 2003,9012,0,1,1954,lecathlon
@@ -3377,18 +3380,6 @@
 
 
 ========================================
-Subject,Gender,Task,MEQ,Time,Date
-756,f,LDT,0,17:07:43,03-25-2002
-
-numCorrect,rawScore,vocabAge,shipTime,readTime
-0,0,0,0,0
-
-presHealth,pastHealth,vision,hearing,firstLang
-7,5,6,7,English
-========================================
-
-
-========================================
 Subject,Gender,Task,MEQ,Time,Date
 756,f,LDT,40,17:10:14,03-25-2002
 
diff -ruN ldt_orig/Data2030.LDT ldt_raw/Data2030.LDT
--- ldt_orig/Data2030.LDT	2009-08-05 16:31:00.000000000 -0500
+++ ldt_raw/Data2030.LDT	2021-11-08 10:13:08.000000000 -0600
@@ -2003,7 +2003,7 @@
 
 
 Univ,Time,Date,Subject,DOB,Education
-6,12:57:53,04-04-2002,763,7282,14
+6,12:57:53,04-04-2002,763,07/02/82,14
 2001,10383,1,1,823,disorderly
 2002,29315,0,1,652,negulate
 2003,37541,1,0,743,udder
diff -ruN ldt_orig/Data2053.LDT ldt_raw/Data2053.LDT
--- ldt_orig/Data2053.LDT	2009-08-05 16:31:00.000000000 -0500
+++ ldt_raw/Data2053.LDT	2021-11-08 08:50:26.000000000 -0600
@@ -2003,7 +2003,7 @@
 
 
 Univ,Time,Date,Subject,DOB,Education
-1,14:17:41,03-17-2002,90,9/27/1981,12
+1,14:17:41,03-17-2002,90,09/27/1981,12
 2001,39103,1,0,673,vocalic
 2002,24223,1,1,633,occupant
 2003,1776,1,0,653,ardor
diff -ruN ldt_orig/Data2058.LDT ldt_raw/Data2058.LDT
--- ldt_orig/Data2058.LDT	2009-08-05 16:31:00.000000000 -0500
+++ ldt_raw/Data2058.LDT	2021-11-08 08:52:09.000000000 -0600
@@ -2003,7 +2003,7 @@
 
 
 Univ,Time,Date,Subject,DOB,Education
-1,16:42:13,04-09-2002,95,01/21/7981,13
+1,16:42:13,04-09-2002,95,01/21/1981,13
 2001,5192,0,1,1870,marpeting
 2002,38853,1,1,1057,versatility
 2003,6749,1,1,1239,comforted
diff -ruN ldt_orig/Data3034.LDT ldt_raw/Data3034.LDT
--- ldt_orig/Data3034.LDT	2009-08-05 16:31:00.000000000 -0500
+++ ldt_raw/Data3034.LDT	2021-10-28 10:25:19.000000000 -0500
@@ -2003,11 +2003,7 @@
 
 
 Univ,Time,Date,Subject,DOB,Education
-5,13:27:30,06-13-2001,626,92073,234
-
-
-Univ,Time,Date,Subject,DOB,Education
-5,13:32:34,06-13-2001,626,02,2
+5,13:27:30,06-13-2001,626,92073,2
 2005,2755,1,1,684,bark
 2006,25794,0,1,1367,penethation
 2007,2380,0,1,2334,apailable
@@ -3381,18 +3377,6 @@
 
 
 ========================================
-Subject,Gender,Task,MEQ,Time,Date
-626,f,LDT,41,15:17:09,04-04-2002
-
-numCorrect,rawScore,vocabAge,shipTime,readTime
-0,0,0,0,1
-
-presHealth,pastHealth,vision,hearing,firstLang
-4,7,5,7,English
-========================================
-
-
-========================================
 Subject,Gender,Task,MEQ,Time,Date
 626,f,LDT,41,15:24:17,04-04-2002
 
diff -ruN ldt_orig/Data3242.LDT ldt_raw/Data3242.LDT
--- ldt_orig/Data3242.LDT	2009-08-05 16:31:00.000000000 -0500
+++ ldt_raw/Data3242.LDT	2021-11-12 15:22:11.000000000 -0600
@@ -2003,7 +2003,7 @@
 
 
 Univ,Time,Date,Subject,DOB,Education
-1,09:07:22,02-06-2003,123,0921983,12
+1,09:07:22,02-06-2003,123,09211983,12
 2001,15639,1,1,621,grateful
 2002,18940,0,0,640,irtersection
 2003,36063,1,1,584,textured
diff -ruN ldt_orig/Data3572.LDT ldt_raw/Data3572.LDT
--- ldt_orig/Data3572.LDT	2009-08-05 16:31:00.000000000 -0500
+++ ldt_raw/Data3572.LDT	2021-10-28 11:14:53.000000000 -0500
@@ -1748,7 +1748,7 @@
 1746,9937,1,1,1575,differed
 1747,12049,0,1,1216,etriching
 1748,11732,0,1,1150,esbraces
-17491750,38492,0,1,1039,upperclustmen
+1750,38492,0,1,1039,upperclustmen
 1751,18144,0,1,1089,indentared
 1752,13749,1,1,684,flake
 1753,25039,0,1,682,oles
